dags/aps/parser.py

```diff
@@ -68,7 +68,7 @@ def __init__(self) -> None:
                 "license",
                 json_path="rights.licenses",
             ),
-            NestedValueExtractor("collections", json_path="HEP.Citeable.Published"),
+            CustomExtractor("collections", extraction_function=lambda x: ["HEP", "Citeable", "Published"]),
```

Comment:
You have to see lambdas as nothing more than a function : If no param is needed you don't put it : 
```python3
lambda: ["HEP", "Citeable", "Published"]
```

Reply:
For now it won't work like this.
I need to change the code here: https://github.com/cern-sis/workflows/blob/main/dags/common/parsing/json_extractors.py#L29-L30

it expects a variable, which at this moment is x in lambda

Reply:
interesting, try to fix it and getting an error:
`TypeError: <lambda>() takes 0 positional arguments but 1 was given`